### PR TITLE
Update README action version

### PR DESCRIPTION
**Pull request by jackton1**
## Description of the change

Updates the version specified in README.md

> Description here
## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by jackton1** [comment]
@mani-rollbar

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -2,8 +2,12 @@ name: CI
 
 on:
   push:
+    paths-ignore:
+      - 'README.md'
```
Comment (mrunalk) [inline .github/workflows/test.yaml:6]: May I know why you need to ignore README?

Comment (jackton1) [reply .github/workflows/test.yaml:6]: Since changes to the README doesn't affect the outcome of the test.

Comment (mrunalk) [reply .github/workflows/test.yaml:6]: But can a change to README affect the outcome of the tests? I don't think tests use README as their input in any way.

**Comment by jackton1** [comment]
@mrunalk @basoko The test's fails due to github secrets not exposed to forks. 

Screen Shot 2021-01-19 at 2 19 07 PM

File: .github/workflows/test.yaml (hunk 1 above)
Comment (jackton1) [reply .github/workflows/test.yaml]: Okay.

**Pull request closed without merging**